apps/console/src/public/deployment.config.json

```diff
@@ -398,9 +398,7 @@
                 }
             },
             "attributeDialects": {
-                "disabledFeatures": [
-                    "attributeDialects.distinct.attribute.profiles"
-                ],
+                "disabledFeatures": [],
```

Comment:
Hope we have sent the corresponding framework PR as well

Reply:
Yes. This is done